src/apml_manager.cpp

```diff
@@ -1102,8 +1102,10 @@ void Manager::harvestUncoreIftDump(
 
         if (ret == OOB_SUCCESS)
         {
-            lg2::info("Socket: {SOCKET},Debug Log ID : {ID} read successful",
-                      "ID", blockId25, "SOCKET", socNum);
+            lg2::info("Socket: {SOCKET},Debug Log ID : {ID} read successful,"
+                      "Block instance : {INST} , Err log length : {LEN}",
+                      "ID", blockId25, "SOCKET", socNum, "INST", resp.instances,
```

Comment:
NIT
```suggestion
                      "SOCKET", socNum, "ID", blockId25, "INST", resp.instances,
```
So that it will be in same order as we print
"Socket: {SOCKET},Debug Log ID : {ID} read successful,"